components/Tabs/src/MDCTabBar.m

```diff
@@ -98,6 +100,9 @@ + (void)initialize {
   [[[self class] appearance] setUnselectedItemTintColor:[UIColor colorWithWhite:1.0 alpha:0.7f]];
   [[[self class] appearance] setInkColor:[UIColor colorWithWhite:1.0 alpha:0.7f]];
   [[[self class] appearance] setBarTintColor:nil];
+
+  id<MDCTabBarIndicatorTemplate> template = [[MDCTabBarUnderlineIndicatorTemplate alloc] init];
+  [[[self class] appearance] setSelectionIndicatorTemplate:template];
```

Comment:
If a property is defined, prefer to use it over the methods.
[[self class] appearance].selectionIndicatorTemplate = template;
Xcode > Edit > Convert > Modernize Objective-c Syntax will do this for you automagically.